iceoryx_utils/include/iceoryx_utils/internal/cxx/variant_queue.inl

```diff
@@ -50,57 +50,48 @@ inline VariantQueue<ValueType, Capacity>::VariantQueue(const VariantQueueTypes t
 }
 
 template <typename ValueType, uint64_t Capacity>
-inline expected<optional<ValueType>, VariantQueueError>
-VariantQueue<ValueType, Capacity>::push(const ValueType& value) noexcept
+optional<ValueType> VariantQueue<ValueType, Capacity>::push(const ValueType& value) noexcept
 {
-    expected<optional<ValueType>, VariantQueueError> ret = error<VariantQueueError>(VariantQueueError::InternalError);
+    optional<ValueType> ret = cxx::nullopt_t();
 
     switch (m_type)
     {
     case VariantQueueTypes::FiFo_SingleProducerSingleConsumer:
     {
-        auto hasSuccess =
+        auto hadSpace =
```

Comment:
I'd prefer to have something like `pushWasSuccessful`. No strong feeling on this